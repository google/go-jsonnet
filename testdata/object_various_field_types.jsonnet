{
	identifier: true,
	"str_double ' \n": true,
	'str_single " \n': true,
	|||
		str_block \n
	|||: true,
	@"str_verbatim_double "" '' \n": true,
	@'str_verbatim_single "" '' \n': true,
	["expr, " + "as complex as you want"]: true,
	local local_field = true,
	assert true
}
