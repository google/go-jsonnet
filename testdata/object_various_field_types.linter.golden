../testdata/object_various_field_types:11:8-26 Unused variable: local_field

	local local_field = true,


